app/views/partials/lib/commands/instances/create.liquid

```diff
@@ -0,0 +1,7 @@
+{% liquid 
+      assign domain = domain | remove_first: "https://" | remove_last: "/"
```

Comment:
what you want to do is to use a build/check/execute functions - https://documentation.platformos.com/get-started/contact-us-tutorial/implementing-business-rules

The `build` will do any necessary normalization - like removing https:// for example.
The `check` will perform any necessary validation - ensure the `pos_cli_token` is not blank, has enough characters etc, the domain is really a domain etc.
And finally, execute will run graphql to store in the db.